mix.exs

```diff
@@ -34,11 +34,11 @@ defmodule RedshiftEcto.MixProject do
 
   defp deps do
     [
-      {:ecto, "~> 2.2"},
-      {:postgrex, "~> 0.13"},
-      {:ecto_replay_sandbox, "~> 1.0.0"},
+      {:ecto_sql, "~> 3.2"},
```

Comment:
Ecto 3 split into `ecto` and `ecto_sql`, `ecto_sql` pulls in both dependencies and contains the adapter code.